wandb/sdk/interface/interface_queue.py

```diff
@@ -47,438 +31,20 @@ def __init__(
         process: BaseProcess = None,
         process_check: bool = True,
     ) -> None:
-        super(InterfaceQueue, self).__init__()
         self.record_q = record_q
         self.result_q = result_q
-        self._process = process
-        self._router = None
-        self._process_check = process_check
-
-        self._init_router()
+        super(InterfaceQueue, self).__init__(
+            process=process, process_check=process_check
+        )
```

Comment:
nit: move to top of function
```suggestion
        super().__init__(
            process=process, process_check=process_check
        )
```